lib/rubocop/sorbet/plugin.rb

```diff
@@ -1,5 +1,10 @@
 # frozen_string_literal: true
 
+rubocop_min_version = Gem::Version.create("1.72.1")
+rubocop_version = Gem::Version.create(RuboCop::Version.version)
+
+return if rubocop_version < rubocop_min_version
+
 require "rubocop"
 require "lint_roller"
```

Comment:
If this is required here, we need to declare a dependency on it. Even if it is just a `gem "lint_roller"` in this file so `lint_roller` is a optional dependency. If Rubocop remove the lint_roller dependecy,  this gem will just fail to load.